### PR TITLE
Include categories and tags in the account/contact export and fix GroupBy ListBuilder problem

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no yes
| New feature? | no yes
| BC breaks? | no yes
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | replaces https://github.com/sulu/sulu/pull/6976
| License | MIT
| Documentation PR | 

#### What's in this PR?

This PR introduces own queries for the grouping selects (Count, Group-Concat).

#### Why?

This will allow to use strict mode in SQL.

### CODE REVIEW

File: src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php [hunk 1]
```diff
@@ -266,30 +261,71 @@ public function execute()
...
+        $select = $this->idField->getSelect();
+        $this->queryBuilder->where($select . ' IN (:ids)')->setParameter('ids', $ids);
+
+        $this->assignParameters($this->queryBuilder);
+
+        $groupResult = $this->queryBuilder->getQuery()->getArrayResult();
+
+        $result = [];
+        for ($i = 0; $i < \count($nonGroupResult); ++$i) {
+            $result[] = \array_merge($nonGroupResult[$i], $groupResult[$i]);
```
Comment (alexander-schranz) [inline src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php]: This look risky as we are not checking if `groupResult` matches to the same id as the `nonGroupResult`.

Comment (wachterjohannes) [reply src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php]: @alexander-schranz your right - have changed it to indexed query! should be safe now

File: src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php [hunk 2]
```diff
@@ -266,30 +261,72 @@ public function execute()
...
+        $this->queryBuilder->where($select . ' IN (:ids)')->setParameter('ids', $ids);
+        $this->queryBuilder->indexBy($this->encodeAlias($this->entityName), $this->idField->getSelect());
+
+        $this->assignParameters($this->queryBuilder);
+
+        $groupResult = $this->queryBuilder->getQuery()->getArrayResult();
+
+        $result = [];
+        foreach ($nonGroupResult as $item) {
+            $result[] = \array_merge($item, $groupResult[$item[$this->idField->getName()]] ?? []);
```
Comment (alexander-schranz) [inline src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php:301]: dumb question where is defined that `$groupResult` is indexed by `id`?

Comment (wachterjohannes) [reply src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php:301]: https://github.com/sulu/sulu/pull/6998/files/de328f5e40aeffefbf39f61098c7f59f51838a9d#diff-d5e86b0228f4f8df3e86e01b2c60a132f15cbb4f24d5cda0f4c8f860c816577bR293

**Review by alexander-schranz — APPROVED** [review]

**Pull request merged**